pkg/daemons/config/types.go

```diff
@@ -124,7 +124,7 @@ func GetArgsList(argsMap map[string]string, extraArgs []string) []string {
 			argsMap[splitArg[0]] = "true"
 			continue
 		}
-		argsMap[splitArg[0]] = splitArg[1]
+		argsMap[splitArg[0]] = strings.Join(splitArg[1:], "=")
```

Comment:
You can do the same with just strings.SplitN(arg, "=", 2)

Reply:
@ibuildthecloud fixed